maquetado/scss/styles.scss

```diff
@@ -150,3 +151,79 @@
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (max-width: 1024px) {
+
+  .body-detail {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .book-info-container {
+    flex-grow: 0;
+    margin-right: 0;
+  }
+
+  .book-title-container {
+    margin-bottom: 0;
+  }
+
+  .book-title {
+    font-size: 20px;
+    margin-right: 10px;
```

Comment:
Same comment. This element has top and bottom margins. Set them to `0` outside this media query and use the title container to set the margin bottom to separate the title from the content